Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/azpy/dcc/maya/utils/maya_materials.py

```diff
@@ -0,0 +1,334 @@
+#!/usr/bin/python
```

Comment:
When organizing the files in the API think about the import namespace:

import azpy.dcc.maya.materials.utils

IMO ^ this is much better